### PR TITLE
Symlink new CLI tools / binaries into usr/local/bin

### PR DESCRIPTION
**Pull request by Kami**
Packages change for https://github.com/StackStorm/st2/pull/3033.

I also noticed we forgot to do that for `st2-check-license` binary so I also did it here.

### CODE REVIEW

**Comment by Kami** [comment]
Note: This will fail until the corresponding st2 PR change is merged in.

**Comment by armab** [comment]
:+1:

File: rake/spec/spec_helper.rb [hunk 1]
```diff
@@ -158,8 +158,13 @@ def spec
 
   shebang = /^#!(?<interpreter>.*?)$/m
   if described_class.content.match(shebang)
-    describe file(Regexp.last_match[:interpreter]) do
-      it { is_expected.to be_file & be_executable }
+    # Note: We skip /usr/bin/env lines
+    interpreter_path = Regexp.last_match[:interpreter]
+    if not Regexp.last_match[:interpreter].start_with?("/usr/bin/env")
```
Comment (armab) [inline rake/spec/spec_helper.rb]: I guess you wanted to write `if not interpreter_path.start_with`

Comment (Kami) [inline rake/spec/spec_helper.rb]: Lol, yeah, I did :)

**Pull request merged**